torchdrivesim/simulator.py

```diff
@@ -593,7 +600,7 @@ def __init__(self, road_mesh: BirdviewMesh, kinematic_model: KinematicModel,
                  internal_time: int = 0, traffic_controls: Optional[Dict[str, BaseTrafficControl]] = None,
                  waypoint_goals: Optional[WaypointGoal] = None,
                  agent_types: Optional[Tensor] = None, agent_type_names: Optional[List[str] ] = None,
-                 npc_controller: Optional[NPCController] = None):
+                 agent_lr: Optional[Tensor] = None, npc_controller: Optional[NPCController] = None):
```

Comment:
```suggestion
                 npc_controller: Optional[NPCController] = None, agent_lr: Optional[Tensor] = None):
```
Keep the original order for backwards compatibility.